File: client/frontend/src/app/shared/models/info.ts
export class Info {
    id: string;
    title: string;
    text: string;
    publishDate: Date;
    unpublished?: Date | undefined;
    status?: string;
    constructor(id: string, title: string, text: string, publishDate: Date, unpublished: Date | undefined, status: string){
        this.id = id;
        this.title = title;
        this.text = text;
        this.publishDate = publishDate;
        this.unpublished = unpublished;
        this.status = status;
    }
}

File: client/viewmodels/Info.cs
using System;

namespace viewmodels
{
    public class Info
    {
        public Guid? Id { get; set; }
        public string Title { get; set; }
        public string Text { get; set; }
        public DateTime? PublishDate { get; set; }
        public DateTime? Unpublished { get; set; }
        public int Version { get; set; }
        public string? Status { get; set; }
    }
}
